src/clj/rems/css/styles.clj

```diff
@@ -366,6 +366,7 @@
                      :color "#ccc"}
                     [:&:hover {:color (util/get-theme-attribute :color4)
                                :background-color "#eee"}]]]
+   [:.flash-message-title {:font-weight :bold}]
```

Comment:
I'm not sure how I feel about this change. The problem I have with it is that the boldface message is slightly distracting to me -- it puts too much emphasis on the message in a situation where I just want to note the message quickly and then move on to do other things on the page.

It also does not seem to be necessary for accessibility: the color contrast is high enough so that small text suffices. See screenshot below (similarly for the red text used for failure messages, not shown here):

![image](https://user-images.githubusercontent.com/49641103/64100651-adf62180-cd74-11e9-8801-c68ae6f40db5.png)

It also introduces a small inconsistency with the warning box, which does not have boldface text:

![image](https://user-images.githubusercontent.com/49641103/64100710-cf570d80-cd74-11e9-8314-e9164b19b5ee.png)

Also, at the very least, the red failure message should have the boldface text, as well. This change only seems to affect the success message.

Reply:
Looking at this more closely, the problem that all flash messages do not have the boldface text seems to be caused by flash-message-title attribute only being set in show-default-success! and show-default-error! helper functions in rems.flash-messages, whereas some flash messages are created by calling show-success! and show-error! functions directly.

Reply:
After getting slightly more familiar with the patch, I now realize where the bold text comes from. In those cases where there is an actual error content in addition to the title, it is good to visually distinguish the title from the rest:

![image](https://user-images.githubusercontent.com/49641103/64103327-a20d5e00-cd7a-11e9-85dd-ac62dbc2e45f.png)

Without bolded text, it looks like this:

![image](https://user-images.githubusercontent.com/49641103/64103369-c0735980-cd7a-11e9-96f1-0ed33eeeef89.png)

From these two, I prefer the former, with the bold text.

Overall, based on this, I'm fine with this change after all. Some messages might need fine-tuning afterwards regarding what is considered a title and what is content, but the idea is fine.